CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ All notable changes to this project are documented in this file.
 - Expose the CoM trajectory computed by the `CentroidalMPC` (https://github.com/ami-iit/bipedal-locomotion-framework/pull/712)
 - Implement python bindings for `CameraBridge` (https://github.com/ami-iit/bipedal-locomotion-framework/pull/695)
 - Implement `constructRDGBSensorClient()` in `RobotInterface` component (https://github.com/ami-iit/bipedal-locomotion-framework/pull/695)
+- Added the ``DistanceTask`` and ``GravityTask`` to the IK (https://github.com/ami-iit/bipedal-locomotion-framework/pull/717)
```

Comment:
Can you move it next to `unrleased`?

Reply:
Done in https://github.com/ami-iit/bipedal-locomotion-framework/pull/717/commits/1d17868cee68fb1eadf04d3b9d87d481bfc07cfa